test/unit/deploy.js

```diff
@@ -1,3 +1,5 @@
+var javascript = require('../../lib/tessel/deployment/javascript');
```

Comment:
This object is already globally available in this file as `deployment.js` (same thing for `deployment.rs` and `deployment.py`)